underscore.js

```diff
@@ -131,6 +132,10 @@
     return result;
   };
 
+  var baseGetPrototype = nativeGetPrototypeOf || function() {
+    return null;
+  };
```

Comment:
By itself, I find this fallback function a bit questionable. I would either expect it to try harder to return a prototype (e.g. by trying to access the `__proto__` property of the argument) or have it named differently in order to signal that it doesn't attempt to replace `Object.getPrototypeOf`, for example `getPrototypeOrNull`.

I realize this function was introduced specifically to facilitate the change in `_.has`, but given the great distance, there is no evident link between the two.